README.md

```diff
@@ -12,18 +12,19 @@ TODO: description
 
 ### Shared Account
 
-#### `POST /api/accounts` - Create an new shared account
+#### `POST /api/accounts` - Create a shared account
 
 **Body**
 
-- `name` _{string}_ - The account owner's name
+- `name` _{string}_ - The shared account owner's name
 - `username` _{string}_ - The user's username
```

Comment:
[nit] The account owner's username

Only mention this as a super small nit since we don't really have a user concept. No need to change ofc, but if so, applies to paths below